shared/js/l10n.js

```diff
@@ -1348,12 +1348,21 @@
   // Getting ready
 
   function negotiate(available, requested, defaultLocale) {
-    if (available.indexOf(requested[0]) === -1 ||
-        requested[0] === defaultLocale) {
+    var supportedLocale;
+    // Find the first locale in the requested list that is supported.
+    for (var i = 0; i < requested.length; ++i) {
```

Comment:
Is there any difference between `++i` and `i++` here?  It seems that in other parts of the code we use post-incrementing for loop counters.